install/threesdk/lib/SDKContainers.py

```diff
@@ -49,34 +56,50 @@ def fill_words():
                 if "." not in identity:
                     identity += ".3bot"
                 self.args.identity = identity
+
             user = self._get_user()
             if not user and not self.args.email:
-                email = self.core.IT.Tools.ask_string("what is your email associated with your identity?")
-                self.args.email = email
-            else:
+                while not self.args.email:
+                    email = self.core.IT.Tools.ask_string("What is the email address associated with your identity?")
+                    if not self._check_email(email):
+                        self.args.email = email
+                    else:
+                        response = self.core.IT.Tools.ask_choices(
+                            "This email is currently associated with another identity. What would you like to do?",
+                            ["restart", "reenter"],
+                        )
+                        if response == "restart":
+                            return False
+            elif user:
                 print("Configured email for this identity is {}".format(user["email"]))
                 self.args.email = user["email"]
+
             if not self.args.words:
-                if user:
-                    fill_words()
-                else:
-                    self.args.words = None
-            if user and self.args.words:
+                fill_words()
```

Comment:
i was making it possible to leave words empty if it's a new user. to they can be generated later on by container install.. now you are forcing the user to enter words...

Reply:
yeah but that caused a recursion for me